spec/rails_helper.rb

```diff
@@ -35,7 +35,7 @@
   config.infer_spec_type_from_file_location!
   config.order = :random
   config.render_views
-  config.include FactoryGirl::Syntax::Methods
+  config.include FactoryBot::Syntax::Methods
 
-  FactoryGirl::SyntaxRunner.send(:include, RSpec::Mocks::ExampleMethods)
+  FactoryBot::SyntaxRunner.send(:include, RSpec::Mocks::ExampleMethods)
```

Comment:
⚠️ **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Lint/SendWithMixinArgument: Use include RSpec::Mocks::ExampleMethods instead of send(:include, RSpec::Mocks::ExampleMethods).